privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -489,7 +489,11 @@ def _ldap_attributes_to_user_object(self, attributes):
             for map_k, map_v in self.userinfo.items():
                 if ldap_k == map_v:
                     if ldap_k == "objectGUID":
-                        ret[map_k] = ldap_v[0]
+                        # An objectGUID should be no list, since it is unique
+                        if type(ldap_v) == basestring:
```

Comment:
I think this won't work, because type("") is ``str`` and type(u"") is ``unicode``. Rather, we should use ``isinstance(ldap_v, basestring)``.

Reply:
OK

Reply:
Done